azkaban-web-server/src/main/java/azkaban/imagemgmt/utils/ImageMgmtConstants.java

```diff
@@ -0,0 +1,7 @@
+package azkaban.imagemgmt.utils;
+
+public interface ImageMgmtConstants {
```

Comment:
Can you please move this interface as class in common Constants file in az-core module?

Reply:
Done.